docker-compose.yml

```diff
@@ -28,6 +28,24 @@ services:
     ports:
       - ${PORT__HEALTHCHECK_METRICS:-7300}:7300
 
+  eigenda-proxy:
+    platform: linux/amd64
+    image: us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/eigenda-proxy:v1.4.1
+    restart: unless-stopped
+    stop_grace_period: 5m
+    entrypoint: /scripts/start-eigenda-proxy.sh
+    volumes:
+      - eigenda-data:/data
```

Comment:
Ah, this is the volume from the previous PR. You can the addition in the previous PR if that makes it easier for you. We'll hopefully merge all of them quickly.

Reply:
I updated this the previous and this pr